src/yea/ytest.py

```diff
@@ -120,10 +124,54 @@ def __init__(self, *, tname: pathlib.Path, yc: "context.YeaContext") -> None:
         self._time_end: Optional[Union[int, float]] = None
         self._permute_groups: Optional[List[Any]] = None
         self._permute_items: Optional[Tuple[Any, ...]] = None
+        self._yearc_list: List[configparser.ConfigParser] = []
+        self._registry: Optional["registry.Registry"] = None
+        self._permute_id: str = ""
 
     def __str__(self) -> str:
         return f"{self._tname}"
 
+    def _add_yearc_list(self, yearc_list: List) -> None:
+        self._yearc_list = yearc_list
+
+    def _add_registry(self, r: "registry.Registry") -> None:
+        self._registry = r
+
+    def _change_yeadoc_path(self, path: pathlib.Path) -> None:
+        self._tname = path
+
+    @property
+    def skip(self) -> bool:
+        self._load()
```

Comment:
nit: i would break it down into multiple function for each type of skip

Reply:
it was going to be a string for each skip type..  but later, did bool for now since that is all we need.